cdap-docs/admin-manual/source/installation/quick-start.rst

```diff
@@ -10,6 +10,9 @@ Installation Quick Start
 
 These instructions will take you from determining your deployment architecture, installing
 CDAP on a Hadoop cluster, through the running of a verification application in CDAP.
+Use these instructions if you aren't using a distribution such as :ref:`Cloudera <cloudera-configuring>`, 
+:ref:`Ambari <ambari-configuring>`, or the :ref:`standalone SDK <standalone-index>`. For the
+`MapR distribution <https://www.mapr.com>`__, we have :ref:`additional instructions <mapr-configuring>` to start with.
```

Comment:
same comment as above:
Use these instructions if you aren't using a cluster manager such as Cloudera Manager or Ambari. For the Mapr distribution, we have additional instructions to start with. These instructions do not apply to the standalone SDK. (or something like that)